Jenkinsfile

```diff
@@ -402,27 +443,27 @@ stage('Default build') {
 					// To that end, escape the '$' when referencing the variables.
 					// See https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#string-interpolation
 					withCredentials([usernamePassword(
-									credentialsId: sonarCredentialsId,
-									usernameVariable: 'SONARCLOUD_ORGANIZATION',
-									passwordVariable: 'SONARCLOUD_TOKEN'
+							credentialsId: sonarCredentialsId,
+							usernameVariable: 'SONARCLOUD_ORGANIZATION',
+							passwordVariable: 'SONARCLOUD_TOKEN'
 					)]) {
 						sh """ \
-								mvn sonar:sonar \
-								-Dsonar.organization=\${SONARCLOUD_ORGANIZATION} \
-								-Dsonar.host.url=https://sonarcloud.io \
-								-Dsonar.login=\${SONARCLOUD_TOKEN} \
-								${helper.scmSource.pullRequest ? """ \
-										-Dsonar.pullrequest.branch=${helper.scmSource.branch.name} \
-										-Dsonar.pullrequest.key=${helper.scmSource.pullRequest.id} \
-										-Dsonar.pullrequest.base=${helper.scmSource.pullRequest.target.name} \
-										${helper.scmSource.gitHubRepoId ? """ \
-												-Dsonar.pullrequest.provider=GitHub \
-												-Dsonar.pullrequest.github.repository=${helper.scmSource.gitHubRepoId} \
-										""" : ''} \
-								""" : """ \
-										-Dsonar.branch.name=${helper.scmSource.branch.name} \
-								"""} \
-						"""
+							mvn sonar:sonar \
+							-Dsonar.organization=\${SONARCLOUD_ORGANIZATION} \
+							-Dsonar.host.url=https://sonarcloud.io \
+							-Dsonar.login=\${SONARCLOUD_TOKEN} \
+							${helper.scmSource.pullRequest ? """ \
+									-Dsonar.pullrequest.branch=${helper.scmSource.branch.name} \
+									-Dsonar.pullrequest.key=${helper.scmSource.pullRequest.id} \
+									-Dsonar.pullrequest.base=${helper.scmSource.pullRequest.target.name} \
+									${helper.scmSource.gitHubRepoId ? """ \
+											-Dsonar.pullrequest.provider=GitHub \
+											-Dsonar.pullrequest.github.repository=${helper.scmSource.gitHubRepoId} \
+									""" : ''} \
+							""" : """ \
+									-Dsonar.branch.name=${helper.scmSource.branch.name} \
+							"""} \
+					"""
```

Comment:
Having fun with automatic formatting aren't you? :)

I must say this one doesn't seem particularly good...

Reply:
🙈👀:wasntme:😃